.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+#### Summary
+<!-- Provide a short summary of your changes -->
+
+#### Description
+<!-- Describe the changes in this PR here and provide some context -->
+
+#### Reviewers
+<!-- The persons assigned to review this PR should mark their item as done when they are happy with this PR to be merged -->
+
+- [ ] Reviewer 1 has approved the PR
+- [ ] Reviewer 2 has approved the PR
```

Comment:
I guess we can remove this section. See https://github.com/sphereio/sphere-product-type-export/pull/4/files#r82736614